codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyHeaders.java

```diff
@@ -150,11 +150,18 @@ private HttpNames() { }
     SpdyHeaders setObject(CharSequence name, Object... values);
 
     @Override
-    SpdyHeaders set(TextHeaders headers);
+    SpdyHeaders set(Headers<? extends CharSequence> headers);
 
     @Override
-    SpdyHeaders setAll(TextHeaders headers);
+    SpdyHeaders setAll(Headers<? extends CharSequence> headers);
 
     @Override
     SpdyHeaders clear();
+
+    /**
+     * {@link #get(CharSequence)} and convert the result to a {@link String}.
+     * @param name the name of the header to retrieve
+     * @return the first header value if the header is found. {@code null} if there's no such header.
+     */
+    String getAsString(CharSequence name);
```

Comment:
I think there was a lot of interest in removing the Conversion routines and simplifying the class hierarchy https://github.com/netty/netty/issues/3655.  Maybe instead of gettting rid of them completely I should put them in a utility class


```java
public static <T> List<String> getAllAsString(Headers<T> headers) {
 ...
}

```

Let me take a stab at this...